reference/config_files/global_conf.rst

```diff
@@ -0,0 +1,383 @@
+
+
+.. important::
+
+    Remember to run that command locally to actually see the latest list because this one may be outdated.
+
+
+Tools configurations
+--------------------
+
+Tools and user configurations allows them to be defined both in the *global.conf* file and in profile files. Profile values will
+have priority over globally defined ones in *global.conf*, and can be defined as:
+
+.. code-block:: text
+
+    [settings]
+    ...
+
+    [conf]
+    tools.microsoft.msbuild:verbosity=Diagnostic
+    tools.microsoft.msbuild:max_cpu_count=2
+    tools.microsoft.msbuild:vs_version = 16
+    tools.build:jobs=10
+
+
+Configuration file template
+---------------------------
+
+Available since: `1.46.0 <https://github.com/conan-io/conan/releases/tag/1.46.0>`_
+
+It is possible to use **jinja2** template engine for *global.conf*. When Conan loads this file, immediately parses
+and renders the template, which must result in a standard tools-configuration text.
+
+  .. code:: jinja
+
+     # Using all the cores automatically
+     tools.build:jobs={{os.cpu_count()}}
+     # Using the current OS
+     user.myconf.system:name = {{platform.system()}}
+
+
+.. note::
+
+    The Python packages passed to render the template are ``os`` and ``platform`` for all platforms and ``distro`` in Linux platforms.
+
+
+Configuration data types
+------------------------
+
+Available since: `1.46.0 <https://github.com/conan-io/conan/releases/tag/1.46.0>`_
+
+All the values will be interpreted by Conan as the result of the python built-in `eval()` function:
+
+.. code-block:: text
+
+    # String
+    tools.microsoft.msbuild:verbosity=Diagnostic
+    # Boolean
+    tools.system.package_manager:sudo=True
+    # Integer
+    tools.microsoft.msbuild:max_cpu_count=2
+    # List of values
+    user.myconf.build:ldflags=["--flag1", "--flag2"]
+    # Dictionary
+    tools.microsoft.msbuildtoolchain:compile_options={"ExceptionHandling": "Async"}
+
+
+Configuration data operators
+----------------------------
+
+Available since: `1.46.0 <https://github.com/conan-io/conan/releases/tag/1.46.0>`_
+
+It's also possible to use some extra operators when you're composing tool configurations in your *global.conf* or
+any of your profiles:
+
+* ``+=`` == ``append``: appends values at the end of the existing value (only for lists).
+* ``=+`` == ``prepend``: puts values at the beginning of the existing value (only for lists).
+* ``=!`` == ``unset``: gets rid of any configuration value.
+
+.. code-block:: text
+    :caption: *myprofile*
+
+    [settings]
+    ...
+
+    [conf]
+    # Define the value => ["-f1"]
+    user.myconf.build:flags=["-f1"]
+
+    # Append the value ["-f2"] => ["-f1", "-f2"]
+    user.myconf.build:flags+=["-f2"]
+
+    # Prepend the value ["-f0"] => ["-f0", "-f1", "-f2"]
+    user.myconf.build:flags=+["-f0"]
+
+    # Unset the value
+    user.myconf.build:flags=!
+
+
+Configuration in your profiles
```

Comment:
Maybe move this section to the "profiles" section, and link it from here.

Reply:
Let's do it when adding the `profiles` section.